### PR TITLE
Enable fips for several 4.14 tests

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -503,29 +506,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disconnected-sdn-rhcos-rhel8-p2-f7
-  cron: 25 20 5,13,20,27 * *
+- as: aws-ipi-disconnected-sdn-rhcos-rhel8-fips-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:509]: hmm, this workflow does not support fips yet, more details are mentioned in https://docs.google.com/spreadsheets/d/1mgPxhb4iUsPQfItdj9rzHGL0-iLY6GNFs5pijIW7G6o/edit#gid=492690660.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -783,27 +788,28 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
-- as: aws-ipi-ovn-ipsec-p1-f4
-  cron: 46 7 2,6,10,14,18,22,27 * *
+- as: aws-ipi-ovn-ipsec-fips-p1-f4
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:791]: I think we can give up the transition to fips mode per the discussion in https://docs.google.com/spreadsheets/d/1mgPxhb4iUsPQfItdj9rzHGL0-iLY6GNFs5pijIW7G6o/edit?disco=AAAAyP2rHTE

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 3]
```diff
@@ -208,27 +208,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-role
```
[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:210]: I checked the initial proposal in https://docs.google.com/spreadsheets/d/1DvzttiDt-mhxCXpZW5MR2vPn1vzYQTZ3xX7T1hi46kc/edit#gid=492690660&fvid=736389283, the install profile is mentioned, but I am not sure why it was missing in our copy version today. Whatever, I saw other versions did transfer it to fips mode.

[the remark this answers is not in the document]
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:210]: To be consistent, I suggest to transfer it to fips mode.

[the remark this answers is not in the document]
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:211]: Added

**Pull request merged**